### PR TITLE
Update itemGroup.json

### PR DESCRIPTION
**Pull request by Chikorita-Lover**
- Remove non-existent item groups
- Add missing item groups
- Add creative categories
- Sort item groups

### CODE REVIEW

File: packages/minecraftBedrock/schema/general/vanilla/itemGroup.json [hunk 1]
```diff
@@ -9,95 +9,85 @@
...
-				"itemGroup.name.bows",
-				"itemGroup.name.totem",
-				"itemGroup.name.natureMisc",
-				"itemGroup.name.sign",
+				"nature",
+				"construction",
+				"items",
+				"equipment",
+				"none",
+				"commands",
```
Comment (Joelant05) [inline packages/minecraftBedrock/schema/general/vanilla/itemGroup.json:17]: These categories are already in https://github.com/bridge-core/editor-packages/blob/main/packages/minecraftBedrock/schema/general/vanilla/creativeCategory.json

Is there somewhere where these categories are valid and not referenced in the schemas?

**Review by Joelant05 — CHANGES_REQUESTED** [review]
See comment on creative categories

File: packages/minecraftBedrock/schema/general/vanilla/itemGroup.json (hunk 1 above)
Comment (Chikorita-Lover) [reply packages/minecraftBedrock/schema/general/vanilla/itemGroup.json:17]: Yes; `parent` of `minecraft:creative_category` item component, and `group` of `minecraft:creative_category` block component

Comment (Joelant05) [reply packages/minecraftBedrock/schema/general/vanilla/itemGroup.json:17]: 👍 I'll sort this out and merge your changes

**Review by Joelant05 — APPROVED** [review]

**Pull request merged**